science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+
+## 6 Differences between urban and rural districts
+
+Finally, we show in the following two figures the incidence-dependent course of the positivity rates for rapid antigen tests and PCR tests depending on the urban or rural character of the districts.
+
+<figure class="figure-img">
+<img src="./ppa_test_result_inzidenz_rat_3_prefix-1_en.png" class="figure-img img-fluid" alt="Figure 17: Relationship between local 7-day incidence and positivity rate after risk notification, smoothed and with confidence interval, in rural and urban districts (RAT)."/>
+<figcaption aria-hidden="true">Figure 17: Relationship between local 7-day incidence and positivity rate after risk notification, smoothed and with confidence interval, in rural and urban districts (RAT).</figcaption>
+</figure>
+
+<figure class="figure-img">
+<img src="./ppa_test_result_inzidenz_pcr_3_prefix-1_en.png" class="figure-img img-fluid" alt="Figure 18: Relationship between local 7-day incidence and positivity rate after risk notification, smoothed and with confidence interval, in rural and urban districts (PCR)."/>
+<figcaption aria-hidden="true">Figure 18: Relationship between local 7-day incidence and positivity rate after risk notification, smoothed and with confidence interval, in rural and urban districts (PCR).</figcaption>
+</figure>
+
+For both types of tests, the difference between red and green warnings (lower risk with risk exposure) and thus the significance of the assessed risk is greater for rural districts than for urban ones.
+
+## 7 Conclusion
+
+The positivity rate of tests after a red warning is always greater for PCR tests in times of low incidence and for rapid antigen tests than the positivity rates after a green warning or at lower risk without a risk exposure. This greater positivity rate does not only depend on local differences in incidence. The differences between the positivity rates are more clearer in rural areas than in urban ones.
+
+A red warning in the Corona‑Warn‑App therefore expresses an increased risk of infection that is not only dependent on the general incidence of infection, even in times of high incidence.
+
+Greater utilisation of the CWA would make the informative value even more independent of incidence and thus even more reliable, as more risk exposures would then be assessed and detected in time.
+
+The data from the [CWA data donation](../2021-10-15-science-blog-4/#3-privacy-preserving-analytics) is a valuable and irreplaceable tool in answering epidemiological questions. Similar analyses for traditional contact person tracing would be desirable.
+
+These are also further proof that the CWA works reliably and fulfils its purpose. Even at this stage of the pandemic – with a high proportion of vaccinated people, cross-immunity and different behavioural patterns – the CWA provides helpful information and is an important tool in assessing personal risk.
+
+## 8 Continued further
```

Comment:
German heading "So geht es weiter" is translated differently across science blog articles. "Continued further" would be yet another new variation. I would suggest to stick with one we already used in Science Blogs [2](https://www.coronawarn.app/en/science/2021-07-08-science-blog-2/), [3](https://www.coronawarn.app/en/science/2021-08-02-science-blog-3/) and [4](https://www.coronawarn.app/en/science/2021-10-15-science-blog-4/)

```suggestion
## 8 What to expect in the future?
```

Reply:
Added in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/4c8e51aead1712b20d50c96a4cd72ac3b0e3bcb5